File: IT/Interfacce grafiche/Esercizi/Java/Esercizio10.java
// https://stackoverflow.com/questions/22162398/how-to-set-a-background-picture-in-jpanel
/* file Esercizio10.java */

import javax.swing.JFrame;
import javax.swing.BoxLayout;
import javax.swing.JLabel;
import javax.swing.ImageIcon;

public class Esercizio10 {
	public static void main(String [] args) {
		JFrame frame = new JFrame();
		frame.setTitle("Immagine");

		ImageIcon img = new ImageIcon("img/image.gif");
		JLabel background = new JLabel(img);
		frame.add(background);

		frame.pack();
		frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
		frame.setVisible(true);
	}
}

File: IT/2/Interfacce grafiche/Esercizi/Python/esercizio10.py
# file esercizio10.py

try:
	from tkinter import Frame, Label, PhotoImage
except ImportError:
	from Tkinter import Frame, Label, PhotoImage

frame = Frame()

img = PhotoImage(file="img/image.gif")
background = Label(frame, image=img)
background.place(x=0, y=0, relwidth=1, relheight=1)
background.pack()

frame.master.wm_title("Immagine")
frame.pack()
frame.mainloop()

